relaybot.py

```diff
@@ -52,7 +53,15 @@ def get(option):
             options["nickServPassword"] = get("nickServPassword")
 
         factory = factory(options)
-        reactor.connectTCP(options['host'], int(options['port']), factory, int(options['timeout']))
+        optionAsBoolean = { "": False, "false": False, "no": False, "true": True, "yes": True }
+        sentinel = object()
+        ssl = options.get('ssl', sentinel)
+        if(sentinel == ssl):

```

Comment:
No need for parenthesis here, and `variable == value` seems preferable, so how about:

    if ssl == sentinel:

If we're given that `options.get()` gives us the same default object, not just something equal to it, we could do:

   if ssl is sentinel:

but I dunno if that's preferable.